cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -110,11 +86,35 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
         .file(mustExist = true, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = true)
         .convert { it.absoluteFile.normalize() }
 
+    private val allowDynamicVersions by option(
```

Comment:
Commit message:

> If an option has a short name, the short name is used for sorting.

Is that a meaningful rule? As all options have at least a long name, it seems more consistent (and easier to remember) to always sort on the long name.

Reply:
The only option affected by was the `-m, --package-manager` option of the analyzer, but it turns out that this isn't the case anymore since I changed the ordering again after writing that commit message. I'll remove the sentence.

Reply:
Done.